src/components/ui/badge.tsx

```
@@ -0,0 +1,36 @@
+import * as React from "react"
```

ESLint

[error] Replace `"react"` with `'react';`

(prettier/prettier)